### PR TITLE
Add math.sqrt() 

### PR DESCRIPTION
**Pull request by ljbevemyr**
Added math.squrt to the library and created some tests.
This solves the problem of being able to use square root in the VOC system.

## PR Checklist:

- [ x] All new features have been tested
- [x] All new features have been documented
- [x ] I have read the **CONTRIBUTING.md** file
- [x ] I will abide by the code of conduct

### CODE REVIEW

File: python/common/org/python/stdlib/math/math.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+@org.python.Module(
+        __doc__ =
+                "TODO"
+)
+public class math extends org.python.types.Module {
+    public math() {
+        super();
+    }
+
+    public static org.python.Object _STRUCT_TM_ITEMS;
```
Comment (rednaxela5950) [inline python/common/org/python/stdlib/math/math.java]: What does this struct do?

File: python/common/org/python/stdlib/math/math.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    @org.python.Attribute
+    public static org.python.Object __file__ = new org.python.types.Str("python/common/python/math.java");
```
Comment (rednaxela5950) [inline python/common/org/python/stdlib/math/math.java]: What does this do? The file `python/common/python/math.java` does not exist. Perhaps it is misplaced?

File: python/common/org/python/stdlib/math/math.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    @org.python.Method(
+         __doc__ = "Return the square root of number",
+        args ={"number"}
+    )
+    public static org.python.Object sqrt(org.python.Object number) {
+        if (number instanceof org.python.types.Float){
+            if(((org.python.types.Float) number).value < 0.0){
+                throw new org.python.exceptions.ValueError("math domain error");
+            }
+            return new org.python.types.Float(Math.sqrt(((org.python.types.Float) number.__float__()).value));
```
Comment (rednaxela5950) [inline python/common/org/python/stdlib/math/math.java:21]: Hard to read. This could be changed into a variable set outside the if statement so it can be used on line 32.

File: python/common/org/python/stdlib/math/math.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        }
+        else if(number instanceof org.python.types.Int){
+            if(((org.python.types.Int) number).value < 0){
+                throw new org.python.exceptions.ValueError("math domain error");
+            }
+            return new org.python.types.Float(Math.sqrt(((org.python.types.Float) number.__float__()).value));
```
Comment (rednaxela5950) [inline python/common/org/python/stdlib/math/math.java:27]: See comment for line 26.

File: python/common/org/python/stdlib/math/math.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        }
+        else if(number instanceof org.python.types.Bool) {
```
Comment (rednaxela5950) [inline python/common/org/python/stdlib/math/math.java:29]: In python booleans are instances of Int: therefore we will never enter this case since we already check for integers in an earlier if statement on row 28.

File: python/common/org/python/stdlib/math/math.java [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+            boolean val = ((org.python.types.Bool) number).value;
+            return new org.python.types.Float(val ? 1 : 0);
+        }
+        else{
+            throw new org.python.exceptions.TypeError("floor() argument must be real number, not "+ number.typeName());
```
Comment (rednaxela5950) [inline python/common/org/python/stdlib/math/math.java]: floor() should be changed to sqrt()

File: tests/stdlib/test_math.py [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+        self.assertCodeExecution("""
+            import math
+            print(math.sqrt("no number"))
+            """)
+
+    @expectedFailure
+    def test_sqrt_negatives(self):
+        self.assertCodeExecution("""
+            import math
+            print(math.sqrt(-1.0))
```
Comment (rednaxela5950) [inline tests/stdlib/test_math.py:20]: add a test case for a negative integer

File: tests/stdlib/test_math.py [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+from ..utils import TranspileTestCase
+
+
+class MathModuleTests(TranspileTestCase):
+
+    @expectedFailure
+    def test_sqrt_no_number(self):
+        self.assertCodeExecution("""
+            import math
+            print(math.sqrt("no number"))
```
Comment (rednaxela5950) [inline tests/stdlib/test_math.py:13]: This is testing a string. It should be `math.sqrt()`.

File: tests/utils.py [hunk 9]
```diff
@@ -76,7 +76,7 @@ def remove_output_dir():
     )
 
     try:
-        out, err = proc.communicate(timeout=30)
+        out, err = proc.communicate(timeout=60)
```
Comment (rednaxela5950) [inline tests/utils.py:79]: What is the reason behind this change?

**Review by rednaxela5950 — CHANGES_REQUESTED** [review]
Nice! Followed coding conventions, but some lines are dangerously close to the strict line limit of 119. One minor bug found. Tests fail.
